wallet-new/src/Cardano/Wallet/API/V1/Daedalus.hs

```diff
@@ -0,0 +1,74 @@
+-- | This module contains the top level API definition for frontend-related
+-- tasks.  The API endpoints presented here are intended for use with the
+-- Daedalus client, and aren't useful for wallets, exchanges, and other users.
+module Cardano.Wallet.API.V1.Daedalus where
+
+-- migrate everything except for import/export
+
+type API =
+    "daedalus"
```

Comment:
@parsonsmatt Ah, interesting change in perspective -- essentially we are moving away from a "kitchen-sink" style of API where everything is collated together, but we are instead keeping "namespace-free" only the "core APIs" which are specific to the wallet state manipulation, and instead cleanly separating deadalus-specific operations into the `daedalus` namespace.

I think it surely make sense. 

Another advantage I can see with this approach is that it will allow us to separate the "let's update daedalus" vs "let's vote or apply an update at the blockchain level", which are two aspects at the moment confused but that in principle we might to keep disjointed. Nice!

My RESTful OCD would make me incline to say that, if possible, we shouldn't use verbs for endpoints -- if these endpoints are going to stay, can we think about anything different?

Reply:
Yeah, that's a good point. We can change it to `v1/internal/update?action=[apply|postponse]` to clean it up